### PR TITLE
NettyGrpcServer -- configure ALPN to enable TLS

### PR DESCRIPTION
**Pull request by ftokarev**
Without ALPN configured, the following exception is thrown when one tries to instantiate NettyChaincodeServer with TLS enabled:

```
java.lang.IllegalArgumentException: ALPN must be enabled and list HTTP/2 as a supported protocol.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:142)
	at io.grpc.netty.shaded.io.grpc.netty.GrpcSslContexts.ensureAlpnAndH2Enabled(GrpcSslContexts.java:260)
	at io.grpc.netty.shaded.io.grpc.netty.NettyServerBuilder.sslContext(NettyServerBuilder.java:300)
	at org.hyperledger.fabric.shim.NettyGrpcServer.<init>(NettyGrpcServer.java:60)
	at org.hyperledger.fabric.shim.NettyChaincodeServer.<init>(NettyChaincodeServer.java:27)
```

### CODE REVIEW

**Comment by ftokarev** [comment]
The build pipeline is failing with errors that are seemingly unrelated to the changes in the PR

File: fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/shim/NettyGrpcServer.java [hunk 1]
```diff
@@ -23,13 +25,14 @@
  */
 public final class NettyGrpcServer implements GrpcServer {
 
-    private static Log logger = LogFactory.getLog(NettyGrpcServer.class);
+    private static final Log logger = LogFactory.getLog(NettyGrpcServer.class);
```
Comment (mbwhite) [inline fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/shim/NettyGrpcServer.java:28]: Not sure when it happened, but the checkstyle rules require this to be LOGGER

**Review by mbwhite — APPROVED** [review]

**Comment by mbwhite** [comment]
@ftokarev thank you so much for this PR - and so sorry that it didn't get any attention until now; it is a essential update, and I've copied the PR to the release-2.2 branch for the release in the next few days.   If you're able to just update the logger->LOGGER and rebase the PR I'llmerge into main. 

Many thanks

**Comment by ftokarev** [comment]
@mbwhite done!

**Review by mbwhite — APPROVED** [review]

**Comment by mbwhite** [comment]
thankyou @ftokarev

**Pull request merged**